--- /dev/fd/63	2024-10-19 11:04:24.668845547 +0900
+++ /dev/fd/62	2024-10-19 11:04:24.669845659 +0900
@@ -315,7 +315,7 @@
 // Examples:
 //
 // copy_command "xclip -selection clipboard" // x11
-// copy_command "wl-copy"                    // wayland
+copy_command "wl-copy"                    // wayland
 // copy_command "pbcopy"                     // osx
 
 // Choose the destination for copied text
@@ -325,7 +325,7 @@
 //   - system (default)
 //   - primary
 //
-// copy_clipboard "primary"
+copy_clipboard "primary"
 
 // Enable or disable automatic copy (and clear) of selection when releasing mouse
 // Default: true
